<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>375</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #323232;
padding-left: 4px;
spacing: 2px;
border: 1px dashed #76797C;
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="BrowseImg">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>200</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
    background-color: #1e1e1e;
    selection-background-color: #1e1e1e;
    selection-color: black;
    padding: 5px;
    border-style: hidden;
	 border: 1px groove #76797C;
   
    border-radius: 2px;
    color: #eff0f1;
}

QPushButton:disabled
{
    background-color: #323232;
    border-width: 1px;
    border-color: #454545;
    border-style: solid;
    padding-top: 5px;
    padding-bottom: 5px;
    padding-left: 10px;
    padding-right: 10px;
    border-radius: 2px;
    color: #454545;
}

QPushButton:focus {
    background-color: #D1DBCB;
    color: black;
}

QPushButton:pressed
{
    color: black;
    background-color: #D1DBCB;
    padding-top: -15px;
    padding-bottom: -17px;
}</string>
    </property>
    <property name="text">
     <string>BROWSE</string>
    </property>
   </widget>
   <widget class="QLabel" name="imagFolder">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>160</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">	background-color: #323232;
	border: none;

    margin-left: 2px;
    margin-right: 2px;
    color: #D1DBCB;




</string>
    </property>
    <property name="text">
     <string>Select Image Folder</string>
    </property>
   </widget>
   <widget class="QLabel" name="excelFile">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>280</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">	background-color: #323232;
	border: none;

    margin-left: 2px;
    margin-right: 2px;
    color: #D1DBCB;




</string>
    </property>
    <property name="text">
     <string>Select Excel File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="browseIxl">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>330</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
    background-color: #1e1e1e;
    selection-background-color: #1e1e1e;
    selection-color: black;
    padding: 5px;
    border-style: hidden;
	 border: 1px groove #76797C;
   
    border-radius: 2px;
    color: #eff0f1;
}

QPushButton:disabled
{
    background-color: #323232;
    border-width: 1px;
    border-color: #454545;
    border-style: solid;
    padding-top: 5px;
    padding-bottom: 5px;
    padding-left: 10px;
    padding-right: 10px;
    border-radius: 2px;
    color: #454545;
}

QPushButton:focus {
    background-color: #D1DBCB;
    color: black;
}

QPushButton:pressed
{
    color: black;
    background-color: #D1DBCB;
    padding-top: -15px;
    padding-bottom: -17px;
}</string>
    </property>
    <property name="text">
     <string>BROWSE</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
